### DISCUSSION TITLE
Unable to do the setup

### DISCUSSION THREAD

**Discussion by AnkitRastogi238** [pluginpal/strapi-plugin-publisher#106] [General]
Hi Everyone,

I have strapi version 4.13.0, I installed the plugin after that in the plugins.js file

 I put the provided code in the readme file after that I build my application but after that too I am unable to see the publisher in the Information sidebar.

What should I do now to setup the plugin?

Can anyone please suggest me the full steps of setup with the code so that I can implement this plugin.

Thanks

**Comment by ComfortablyCoding** [comment]
In `config/plugins.js` you place the following 
```
module.exports = ({ env }) => ({
  // ..
	'publisher': {
		enabled: true,
		config: {},
	},
	// ..
});
```
Aside from that nothing else is required. 

Sometimes a clean rebuild is required before the plugin will show. Instructions on how to do so are in the [note at the end of the usage instructions](https://github.com/ComfortablyCoding/strapi-plugin-publisher#usage
